test.py

```diff
@@ -0,0 +1,44 @@
+import ecusisUtils
```

Comment:
This file is fine, but better to write unit tests (see https://docs.python.org/2/library/unittest.html). Not a priority.